heron/common/src/cpp/config/topology-config-vars.cpp

```diff
@@ -39,6 +39,11 @@ const sp_string TopologyConfigVars::TOPOLOGY_ENABLE_MESSAGE_TIMEOUTS =
 const sp_string TopologyConfigVars::TOPOLOGY_ENABLE_ACKING = "topology.acking";
 const sp_string TopologyConfigVars::TOPOLOGY_CONTAINER_CPU_REQUESTED = "topology.container.cpu";
 const sp_string TopologyConfigVars::TOPOLOGY_CONTAINER_RAM_REQUESTED = "topology.container.ram";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_ENABLED = "topology.stateful.enabled";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_CHECKPOINT_INTERVAL_SECONDS =
+                            "topology.stateful.checkpoint.interval.seconds";
+const sp_string TopologyConfigVars::TOPOLOGY_STATEFUL_START_CLEAN = "topology.stateful.start.clean";
+const sp_string TopologyConfigVars::TOPOLOGY_EXACTLYONCE_ENABLED = "topology.exactlyonce.enabled";
```

Comment:
It's set in the topology config: https://github.com/twitter/heron/blob/master/heron/api/src/java/com/twitter/heron/api/Config.java#L472

I think we need to clarify the relations of acking and exactly once and educate our users to set them properly. 

Also, I'll suggest if a topology is set to exactly once, then this semantic will override the acking setting.